src/metrics/singleAuthor.ts

```diff
@@ -3,35 +3,26 @@ import type { PointLegendData } from "~/components/legend/PointLegend"
 import { PointInfo } from "~/components/legend/PointLegend"
 import type { AuthorshipType, MetricCache } from "./metrics"
 
-export function setDominanceColor(blob: HydratedGitBlobObject, cache: MetricCache, authorshipType: AuthorshipType) {
-  const dominatedColor = "#b91c1c"
-  const defaultColor = "#fecaca"
-  const nocreditColor = "#14b8a6"
-
+export function setDominanceColor(blob: HydratedGitBlobObject, cache: MetricCache, authorshipType: AuthorshipType, authorColors: Map<string, `#${string}`>) {
+  const multipleAuthorsColor = "#e0e0e0"
```

Comment:
Would it make sense to move these constants to the `consts.ts` or what it is called?

Reply:
We can do that at some other point :)
we are not very consistent with putting constants in there